### ISSUE TITLE
[Bug] ShellContent.IsVisible issue on Android

### ISSUE THREAD

**Issue by aktxyz** [xamarin/Xamarin.Forms#11869]
### Description

ShellContent items on Android not responding to IsVisible binding on Android (ios is fine).

### Steps to Reproduce

1. see git repo for sample to reproduce 
2. https://github.com/aktxyz/SampleApp_Xamarin_PossibleShellItemVisibleIssue
3. 

### Expected Behavior

Android show behave the same as IOS, ie the ShellContent items should hide/show based on the binding.

### Actual Behavior

ShellContent items do not hide

### Basic Information

- Version with issue: xamarin  v4.8.0.1269

### Reproduction Link

https://github.com/aktxyz/SampleApp_Xamarin_PossibleShellItemVisibleIssue

### Workaround

did not find one

**Comment by samhouts** [comment]
closed by #11885

**Comment by jackbond** [comment]
Dunno why this was closed. I've attached what I do below, and when IsChessEnabled changes, the Tab is hidden, but the link in the flyout remains visible. I was seeing this in the latest 4.8, so I upgraded to 5 preview 6, and saw the same behavior.

```
<Tab IsVisible="{Binding IsChessEnabled}">
	<ShellContent IsVisible="{Binding IsChessEnabled}" />
</Tab>
```
